.github/workflows/python-cm-integ-tests.yml

```diff
@@ -67,4 +67,15 @@ jobs:
         pip list
         echo "$GITHUB_WORKSPACE" >> $GITHUB_PATH
         pytest -v test/
-        
+
```

Comment:
Should also delete the IS_CI env variables from Java, Python, and any others.

Reply:
Oh good catch, I missed that. I think only Java and Python ended up merging a version of this but I'll check for other instances of this variable.

Reply:
Should be addressed by [da6531e](https://github.com/aws-samples/aurora-dsql-samples/commit/da6531ef2c0a994fbef5f257805b8ea9322de7f1)